### PR TITLE
chore: replace weth and wbtc addresses

### PR DESCRIPTION
**Pull request by eduardopelitti**
- Update weth and wbtc addresses in overrides, vetted and listed token lists
- Generate kovan tokenlist.json
~- Add .DS_Store files to gitignore~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.DS_Store
+
```
Comment (andyesp) [inline .gitignore:1]: Why did you remove this line?

**Review by andyesp — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (eduardopelitti) [reply .gitignore:1]: 🤦

**Review by andyesp — APPROVED** [review]

**Review by cfranchella — APPROVED** [review]
